requirements-test.txt

```diff
@@ -1,9 +1,9 @@
 # Supported models
-scikit-learn==0.22.2.post1
+scikit-learn==0.23.0
 xgboost==1.0.2
 lightgbm==2.3.1
 statsmodels==0.11.1
-git+git://github.com/scikit-learn-contrib/lightning.git@b96f9c674968496e854078163c8814049a7b9f43
+git+git://github.com/scikit-learn-contrib/lightning@refs/pull/142/head
```

Comment:
https://github.com/scikit-learn-contrib/lightning/pull/142

I don't believe that that my PR will be merged (at least soon), but this trick allows us to install the latest scikit-learn with lightning at Travis and run all our tests. At the same time, users are still able to install latest `master` of lightning with old scikit-learn (0.23.0) and not face any problems with a such environment setup.

Reply:
Wow! They merged it! 🎉